init.vim

```diff
@@ -1091,3 +1091,11 @@ hi CursorLine ctermbg=Black cterm=bold guifg=white guibg=yellow gui=bold
 hi! Normal ctermbg=NONE guibg=NONE
 
 colorscheme solarized
+
+" Congwen
+nmap <Leader>py <Plug>(Prettier)
```

Comment:
I wouldn't recommend `<Leader>py`, `<Leader>p` is already bound:
```
nnoremap <Leader>p mp"_diwP`p
```
What it does is pastes over the cursor's current word and set marker `p`
`<Leader>y` yanks the cursor's current word and set marker `y`.

Adding the `<Leader>py` will add a delay to the paste.